### DISCUSSION TITLE
APU bleed and EXT PWR stays on

### DISCUSSION THREAD

**Discussion by MStallen** [flybywiresim/aircraft#8781] [General]
If I switch the APU (master) off the the APU bleed stays on (without a fault) is that correct?
If you do a pushback without disconnecting EXT PWR the switch stays on (without a fault) is that correct?

**Comment by MStallen** [comment]
Got the answer on Discord

**Reply by alepouna** [reply]
Quoting the answer from Discord for future reference (from slein15)
> That's correct, there's a cool down period of something like 2 min.
> That's an msfs problem. Msfs shows a power connection as long as you're close enough to the stand.
